src/components/customerService/noticeDetail/content/NoticeDetailContent.styled.ts

```
@@ -0,0 +1,64 @@
+export const Admin = styled.span`
+  vertical-align: middle;
+  font-size: 1.1rem;
+`;
+
+export const InfoWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+`;
+
+export const Date = styled.span`
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)